web/util/date.ts

```diff
@@ -30,6 +26,22 @@ export const formatTimeDuration = (start: Date, duration: number) => {
   return `PT${hours ? hours + 'H' : ''}${minutes ? minutes + 'M' : ''}`;
 };
 
+const differingMeridiem = (d1: Date, d2: Date) =>
+  d1.getHours() >= 12 !== d2.getHours() >= 12;
+
+export const formatTimeRange = (
+  start: Date,
+  duration: number,
+  timezone: string
+) => {
+  const end = addMinutes(start, duration);
+  return `${formatTime(
+    start,
+    timezone,
+    differingMeridiem(start, end)
+  )}–${formatTime(end, timezone)}`;
```

Comment:
After the rewrite in 97d3dc91be877398e809fd2d48d4f42cf111065f, this doesn't show AM/PM at the end anymore.
```suggestion
  )}–${formatTime(end, timezone, true)}`;
```